jbd2: fix kernel-doc notation

From: Randy Dunlap <randy.dunlap@oracle.com>

Fix kernel-doc notation in jbd2.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
---
 fs/jbd2/journal.c     |    5 +++--
 fs/jbd2/transaction.c |    3 ++-
 2 files changed, 5 insertions(+), 3 deletions(-)

Index: linux-2.6.25-rc9/fs/jbd2/journal.c
===================================================================
--- linux-2.6.25-rc9.orig/fs/jbd2/journal.c	2008-04-14 11:11:31.000000000 -0700
+++ linux-2.6.25-rc9/fs/jbd2/journal.c	2008-04-14 11:12:30.000000000 -0700
@@ -1006,13 +1006,14 @@ fail:
  */
 
 /**
- *  journal_t * jbd2_journal_init_dev() - creates an initialises a journal structure
+ *  journal_t * jbd2_journal_init_dev() - creates and initialises a journal structure
  *  @bdev: Block device on which to create the journal
  *  @fs_dev: Device which hold journalled filesystem for this journal.
  *  @start: Block nr Start of journal.
  *  @len:  Length of the journal in blocks.
  *  @blocksize: blocksize of journalling device
- *  @returns: a newly created journal_t *
+ *
+ *  Returns: a newly created journal_t *
  *
  *  jbd2_journal_init_dev creates a journal which maps a fixed contiguous
  *  range of blocks on an arbitrary block device.
Index: linux-2.6.25-rc9/fs/jbd2/transaction.c
===================================================================
--- linux-2.6.25-rc9.orig/fs/jbd2/transaction.c	2008-04-14 11:11:46.000000000 -0700
+++ linux-2.6.25-rc9/fs/jbd2/transaction.c	2008-04-14 11:12:30.000000000 -0700
@@ -1462,7 +1462,8 @@ int jbd2_journal_stop(handle_t *handle)
 	return err;
 }
 
-/**int jbd2_journal_force_commit() - force any uncommitted transactions
+/**
+ * int jbd2_journal_force_commit() - force any uncommitted transactions
  * @journal: journal to force
  *
  * For synchronous operations: force any uncommitted transactions
